File: common/src/main/java/dev/number6/CommonModule.java
package dev.number6;

import dev.number6.comprehend.dagger.ComprehensionServiceModule;
import dev.number6.db.dagger.DatabaseConfigurationPortModule;
import dev.number6.db.dagger.DatabasePortModule;
import dev.number6.db.dagger.DynamoDBMapperModule;
import dagger.Module;

@Module(includes = {DynamoDBMapperModule.class,
        DatabasePortModule.class,
        DatabaseConfigurationPortModule.class,
        ComprehensionServiceModule.class})
public class CommonModule {
}

File: common/src/main/kotlin/dev/number6/CommonModule.kt
package dev.number6

import dagger.Module
import dev.number6.comprehend.dagger.ComprehensionServiceModule
import dev.number6.db.dagger.DatabaseConfigurationPortModule
import dev.number6.db.dagger.DatabasePortModule
import dev.number6.db.dagger.DynamoDBMapperModule

@Module(includes = [DynamoDBMapperModule::class,
    DatabasePortModule::class,
//    DatabaseConfigurationPortModule::class,
    ComprehensionServiceModule::class])
class CommonModule 